workspace/TS100/Core/BSP/Pine64/FreeRTOSConfig.h

```diff
@@ -57,7 +55,7 @@ extern uint32_t SystemCoreClock;
 
 /* Software timer related definitions. */
 #define configUSE_TIMERS 0
-#define configTIMER_TASK_PRIORITY (configMAX_PRIORITIES - 1) //Tmr_svc ç‹¬å� æœ€é«˜ä¼˜å…ˆçº§
+#define configTIMER_TASK_PRIORITY3
```

Comment:
Did i spot a missing whitespace in front of the 3?

Reply:
I wonder what's the reason behind changing `(configMAX_PRIORITIES - 1)` to `3` (as previously it was more clear that timer task should have priority 1 below highest.

Reply:
Not entirely sure, but at this point of trying to get something out before 🎄 , deferring to the nuclei settings.